docs/docs/command-line-flags.md

```diff
@@ -47,6 +47,7 @@ The core functionality flags can be also set by environment variable `MARATHON_O
     - "vips" can be used to enable the networking VIP integration UI.
     - "task\_killing" can be used to enable the TASK\_KILLING state in Mesos (0.28 or later)
     - "external\_volumes" can be used if the cluster is configured to use external volumes.
+    - "mesos_healthchecks" can be used if ti enable the Mesos HealthChecks (Mesos 1.1 or later).
```

Comment:
I don't know why it's done that way. Probably to prevent changing font _style_. When you take a look at [rendered html it looks the same](https://github.com/janisz/marathon/blob/ed340f969186ae99d43ad10cc42aadeae2997980/docs/docs/command-line-flags.md#optional-flags).
IMO this should be wrapped in code format (\`) not double quotes (`"`)